homepage.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask 
+app = Flask(__name__)
+
+@app.route("/")
+def home():
+    return "<h1>Price Aggrigator</h1>"
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Comment:
Can you add a new line here at the bottom? GitHub tends to get mad if there isn't an extra line.